JetNews/gradle/libs.versions.toml

```diff
@@ -28,9 +28,9 @@ compileSdk = "33"
 compose = "1.3.0-rc01"
 compose-compiler = "1.3.2"
 # @pin to be updated manually - check JetChat tests
-compose-material3 = "1.0.0-beta03"
+compose-material3 = "1.0.0-rc01"
 # @pin to be updated manually - check JetChat tests
```

Comment:
should we remove the disclaimer?

Reply:
I'm not sure who added it but I would think so